{
  "cveId": "CVE-2022-36291",
  "eventName": "New CVE Received",
  "cveChangeId": "AACE4CFE-3A8E-4BE9-ABE9-3B26824118C9",
  "sourceIdentifier": "secure@intel.com",
  "created": "2025-05-28T19:15:29.463",
  "details": [
    {
      "action": "Added",
      "type": "Description",
      "newValue": "Rejected reason: This CVE ID has been rejected or withdrawn by its CVE Numbering Authority because it is Unused"
    }
  ]
}